docs/design/dab-configure.md

```diff
@@ -0,0 +1,117 @@
+# Design Document: `dab configure` Command
+
+## Overview
+
+The `dab configure` command is designed to simplify updating config properties outside of the **entities** section. This document outlines the design, functionality, and implementation details of the `dab configure` command.
+
+## Objectives
+
+- Add support to CLI to configure the `data-source` and `runtime` sections of the runtime config.
+
+- Ensure configurations are validated before being applied.
+
+## Command Syntax
+
+```sh
+
+dab configure [options]
+
+```
+
+### Options
+
+Some of the supported options:
+- `--config `: Specify the configuration file path.
+- `--runtime.graphql.depth-limit`: Max allowed depth of the nested query.
+- `--data-source.database-type`: Type of database to connect to.
+- `--data-source.options.schema`: Schema path for Cosmos DB for NoSql.
```

Comment:
Maybe it can be made more clear that you can change the all the properties inside 'runtime' and 'data-source', at first it gave me the impression that only some options where available